concent-builder/uninstall-tiller.yml

```diff
@@ -0,0 +1,29 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:     Check if charts exist
+          shell:    '[ -z "$(helm list --short --tiller-namespace tiller)" ] && echo "Empty"'
+          register: result
```

Comment:
Why not put the output from `helm list --short --tiller-namespace tiller` in a variable and check that in `when`? You should not have to resort to Bash conditionals when you're using Ansible and can make Python conditions.

If you did it like this you could also use it in the command instead of calling helm again below.

Reply:
Done.